storage/transfermanager/downloader.go

```diff
@@ -750,6 +756,10 @@ type DownloadDirectoryInput struct {
 	// The directory will be created if it does not already exist. Required.
 	LocalDirectory string
 
+	// StripPrefix is a prefix to be removed from the path of an object on
```

Comment:
Clarify that it's the path of the local file for download rather than the object in GCS.